modules/nodes-pods-vertical-autoscaler-configuring.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-vertical-autoscaler.adoc
+
+[id="nodes-pods-vertical-autoscaler-configuring_{context}"]
+= Using the Vertical Pod Autoscaler
+
+You can use the Vertical Pod Autoscaler Operator (VPA) by creating a Vertical Pod Autoscaler Custom Resource (CR). The CR determines which Deployment or ReplicaSet it should analyze and determines the actions the VPA should take with the Pods associated with that Deployment or ReplicaSet. 
+
+.Procedure
+
+To create a VPA Custom Resource for a specific controller, either a Deployment or ReplicaSet:
```

Comment:
I haven't tested it, but I'm pretty sure VPA can work with pods from any controller (Deployment and ReplicaSet, but also DaemonSet, CronJob, DeploymentConfig, ReplicationController, etc.) so I don't know whether to call that out, but I don't think we want to make it sound like it's limited to rs and deployments.

I'll do some testing to make sure.

Reply:
@joelsmith I tried each of the object types in the console under Workloads that spawn pods: Deployment, Deployment Config, Statefulset, Job, DaemonSet, ReplicaSet, or ReplicationController. I used the recommender. All generated recommendations except for Job (which makes sense as the job completed before the VPA could get its info, I assume) and DeplotmentConfig. For the DC, and I tried a few, I got:
```
message: 'Cannot read targetRef. Reason: Unhandled targetRef apps/v1 / DeploymentConfig
      / example, last error no matches for kind "DeploymentConfig" in group "apps"'
```
Does this message make sense?

Reply:
@weinliu Can you comment on which controllers the VPA works with?

Reply:
Deployment verified to be working